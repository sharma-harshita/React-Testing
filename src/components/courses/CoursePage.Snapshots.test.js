import React from 'react'
import CoursePage from './CoursesPage'
import renderer from 'react-test-renderer';

it ('check CoursePage', ()=>{
    const value = renderer.create(<CoursePage/>)

    expect (value).toMatchSnapshot()
})