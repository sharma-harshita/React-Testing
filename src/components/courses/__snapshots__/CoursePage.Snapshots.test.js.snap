// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check CoursePage 1`] = `
<h2>
  Courses
</h2>
`;
